tests/integ_tests/ephemeral_server_tests.rs

```diff
@@ -30,45 +35,6 @@ async fn temporal_cli_fixed() {
     server.shutdown().await.unwrap();
 }
 
-#[tokio::test]
-async fn temporalite_default() {
-    let config = TemporaliteConfigBuilder::default()
-        .exe(default_cached_download())
-        .build()
-        .unwrap();
-    let mut server = config.start_server().await.unwrap();
-    assert_ephemeral_server(&server).await;
-    server.shutdown().await.unwrap();
-}
-
-#[tokio::test]
-async fn temporalite_fixed() {
-    let config = TemporaliteConfigBuilder::default()
-        .exe(fixed_cached_download("v0.2.0"))
-        .build()
-        .unwrap();
-    let mut server = config.start_server().await.unwrap();
-    assert_ephemeral_server(&server).await;
-    server.shutdown().await.unwrap();
-}
-
-#[tokio::test]
-async fn temporalite_shutdown_port_reuse() {
```

Comment:
Looks like we don't have an this specific test for cli. Would it be pertinent to repurpose it rather than delete it?

Reply:
We never have had it, so for CLI we've never confirmed port reuse, only Temporalite. I was hoping not to have to address new requirements (e.g. confirming CLI allows port reuse post shutdown) as part of this smaller issue, but alas I suppose I can.

Reply:
Fixed